web/stories/Molecules/Button/Calendar.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { CalendarEventItem } from "@/components/filings-calendar/CalendarEventItem";
+import { defaultDateFormat, parseDateWithFormat } from "@businessnjgovnavigator/shared";
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+export default {
+  title: "Molecules/Button/Calendar",
+  component: CalendarEventItem,
+  decorators: [(Story) => <div className="width-mobile">{Story()}</div>],
```

Comment:
[pebble/sand] The styling on line 19 can be applied here `<div style={{ width: "212px", height: "54px" }}>` or remove this line altogether

The decorator key allows you to apply global styling to each individual story.

Reply:
Neto, didn't know about that key. But very cool, placed in decorator for calendar and removed the decorator key from all the other button stories.